packages/graph/hash_graph/lib/graph/src/ontology/property_type.rs

```diff
@@ -39,10 +39,10 @@ pub enum PropertyTypeQueryPath {
     /// ```
     ///
     /// In addition to specifying the version directly, it's also possible to compare the version
-    /// with a `"latest"` parameter, which will only match the latest version of one
+    /// with a `"latest"` parameter, which will only match the latest version of this
```

Comment:
Entity/data type use `the` here :)
```suggestion
    /// with a `"latest"` parameter, which will only match the latest version of the
```